packages/tools/tests/test/visualization/config.json

```diff
@@ -1256,7 +1256,7 @@
         },
         {
             "title": "Glow layer and LODs",
-            "playgroundId": "#UNS6ZV#4",
+            "playgroundId": "#UNS6ZV#9",

```

Comment:
Adding the flag to the creation options I mentioned above would allow us not to modify the PGs, but to update the test code instead. When all the code is ported to WGSL, we'll simply need to remove this flag from the test code. Otherwise, we'll have to track the PGs that have been modified in order to restore them later.

Reply:
Fixed!